.gitignore

```diff
@@ -0,0 +1,6 @@
+manuscript/manuscript.aux
+manuscript/manuscript.dvi
+manuscript/manuscript.log
+manuscript/manuscript.out
+manuscript/manuscript.pdf
```

Comment:
Do we want to ignore the rendered pdf?  Might be nice to include it with the repo.

Other than this comment, the rest looks good to me.